app/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,8 @@
+.usa-alert--info-important {
+  @include u-bg('accent-cool-darker');
+  color: color('white');
+
+  &::before {
+    background-image: url('alert/info-white.svg');
```

Comment:
You'll need to add this image file from my previous comment at https://github.com/18F/identity-idp/pull/7216#discussion_r1004942291

Reply:
Icon added.

Reply:
Did you want the icon to be white?